md/release-notes.md

```diff
@@ -364,6 +364,22 @@ Please use the Tomcat bundle instead, or see the [Custom Deployment](deploy-bund
 ## Bug fixes
 ### Fixes in Documentation
 * BS-19323 Bonita documentation is not clear enough about PostgreSQL versions supported
+* BST-151 Default outgoing transition not executed when  there is more than one outgoing transition from inclusive gateway
+* BST-269 Document workaround for macos - ssh issue
+* BST-171 Can't share a project on github using HTTPS and double authentication factor
+* BST-341 On Mac, Studio does not start if recent Java versions installed
+
+### Fixes in Bonita 7.9.2 (2019-08-01)
+#### Fixes in Engine component
+* BS-19299 GET REST API connector raised an exception if answer HTTP code is 204 or 404
+
+#### Fixes in Studio component
+* BST-345 Open an invalid access control file leads to an NPE instead of displaying the `error view`
+* BST-349 The option to initialize a business data using a Query is not always available
+* BST-365 BS-19299 - GET REST API connector raised an exception if answer HTTP code is 204 or 404
```

Comment:
Duplication, there were two tickets because I was afraid to forget to integrate the new connector version in the Studio. 
I suggest you to write something like 'Bonita Studio now uses REST API connector 1.0.6`

Reply:
thanks for noticing